### ISSUE TITLE
Improve acessibility

### ISSUE THREAD

**Issue by Rarst** [Rarst/laps#5]
The current markup produced by the plugin uses empty divs in bootstrap progress bar components.

The accessibility should be improved by adding `.screen-reader-text` content to identify timelines and provide content of the events.

Also the keyboard-only navigation need to be looked into. At the moment menu entry is being skipped on tabbing through the toolbar.

**Comment by Rarst** [comment]
I had added screen reader text in https://github.com/Rarst/laps/commit/274d294a65de0709b81ae9e3f24bc6a251f81aeb but unsure what's up with keyboard.

I suspect item is skipped because WP adds `.ab-empty-item` since top level item doesn't _link_ to anything, though it still has popup?..

**Comment by clovepod** [comment]
I think it's not tabbable because there's no <a> element in it. You might be able to make it tabbable by adding a tabindex=0 to the div (if you can, and if adding an <a> isn't an option).

**Comment by Rarst** [comment]
There is _text_ in it?.. Not much clue how keyword stuff works. I think "regular" elements tab through because they are links (actionable of sorts?).

What should be tabbable menu element which is not a link?

**Comment by clovepod** [comment]
i'm sorry, github ate part of my comment although maybe you got what i was trying to say.  there is no a (link) element in it. I believe the only html elements that can receive tab focus are links and form elements, although you can force an element to become tabbable by adding a tabindex attribute.  a tabindex of 0 will not re-order the tab focus that would naturally occur.

http://webaim.org/techniques/keyboard/tabindex

to make it more semantic, you could experiment with making it a button element, which is tabbable and would indicate that an action will occur.

to *really* get closer to WCAG AA (which is what most people shoot for), it shouldn't change on mouseover, but I think just making the element tabbable and more semantic would be sufficient. 

i didn't have the sound turned on when i ran it through VoiceOver so i don't know if there was an audible indication of the visual change that occurs on mouseover, I can check that later if you're curious and don't have a way to check.
